sdc/tests/test_series.py

```diff
@@ -1322,7 +1272,26 @@ def test_impl(A, n):
                     S = pd.Series([2, 4, 6, 6, 3], index)
                     pd.testing.assert_series_equal(hpat_func(S, n), test_impl(S, n))
 
-    @unittest.expectedFailure
+    '''
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#string-quotes): For triple-quoted strings, always use double quote characters to be consistent with the docstring convention in PEP 257.

Please fix it here and in some places below.

Reply:
Sure, changed in all places.